onnx/defs/logical/old.cc

```diff
@@ -6,6 +6,99 @@
 using namespace ONNX_NAMESPACE;
 
 namespace ONNX_NAMESPACE {
+
+inline void unaryLogicalOpInference_opset12(InferenceContext& ctx) {
```

Comment:
Is this function used, or is it a copy-paste error?

Reply:
removed.